aria/modeling/service_template.py

```diff
@@ -33,7 +33,8 @@
 from ..parser import validation
```

Comment:
I don't understand. What does this have to do with this line in the code? Where in the commit message? We have limited our commit messages to one line with reference to the JIRA for more information. I know you tend to add more info, but the rest of us don't normally.